src/lib/Preview.js

```diff
@@ -1009,6 +1010,21 @@ class Preview extends EventEmitter {
                 this.enableViewers(viewerName);
             }
         });
+
+        // Disable office online if browser is IE
+        this.forceDisableOfficeOnline();
+    }
+
+    /**
+     * Adds Office viewer to the disabled viewer list if browser is IE
+     * @returns {void}
+     */
+    forceDisableOfficeOnline() {
+        const isIE = this.browserInfo.name === BROWSERS.INTERNET_EXPLORER;
+
+        if (isIE) {
```

Comment:
Can we move this logic to OfficeLoader? We have other conditions that do the same thing there.